papyrus/xsd.py

```diff
@@ -64,6 +64,13 @@ def __init__(self,
         self.include_foreign_keys = include_foreign_keys
         self.sequence_callback = sequence_callback
 
+    def add_column_app_infos(self, tb, column):
+        if column.info.get('readonly'):
+            with tag(tb, 'xsd:annotation'):
+                with tag(tb, 'xsd:appinfo'):
+                    with tag(tb, 'readonly', {'value': 'true'}):
```

Comment:
Just wondering if the resulting XSD document is valid with the readonly element having no prefix?